Tests/BodegaTests/ObjectStorageTests.swift

```diff
@@ -1,12 +1,91 @@
 import XCTest
 @testable import Bodega
 
-final class ObjectStorageTests: XCTestCase {
+// Testing an ObjectStorage instance that's backed by a SQLiteStorageEngine
+final class SQLiteStorageEngineBackedObjectStorageTests: ObjectStorageTests {
 
-    private var storage: ObjectStorage!
+    override func setUp() async throws {
+        storage = ObjectStorage(
+            storage: SQLiteStorageEngine(directory: .temporary(appendingPath: "SQLiteTests"))!
```

Comment:
same question here (and others using `directory: .temporary`) re: test pollution